src/main.js

```diff
@@ -37,6 +37,31 @@ function createWindow() {
     icon: path.join(__dirname, 'assets/icons/png/256x256.png'),
   });
 
+  if (process.env.NODE_ENV === 'development') {
+    const {
+      default: installExtension,
+      REACT_DEVELOPER_TOOLS,
+      REACT_PERF,
+    } = require('electron-devtools-installer');
+
+    require('electron-debug')({ showDevTools: true });
+
+    const ChromeLens = {
+      id: 'idikgljglpfilbhaboonnpnnincjhjkd',
```

Comment:
I'm not familiar with ChromeLens, what's the `id` for?

Reply:
That's the id of the extension : https://chrome.google.com/webstore/detail/chromelens/idikgljglpfilbhaboonnpnnincjhjkd <- last part of the url

Reply:
TIL 👍

Reply:
Could you please add this info as a comment in the code? Would be great to document that :)